components/Home/Mech/index.jsx

```diff
@@ -25,7 +25,7 @@ const WEBSOCKET_PROVIDER = process.env.NEXT_PUBLIC_GNOSIS_WEB_SOCKET;
 const { Title } = Typography;
 
 // const filterOption = { fromBlock: 28127133, toBlock: 'latest' };
-const filterOption = { fromBlock: 30157493, toBlock: 'latest' }; // TODO: add pagination
+const filterOption = { fromBlock: 30207493, toBlock: 'latest' }; // TODO: add pagination
```

Comment:
Maybe it would be better to calculate this dynamically? e.g. currentBlock - 5000. Otherwise the performance is going to be degraded every few days and we'll need to constantly update